router/router.go

```diff
@@ -1073,14 +1109,19 @@ func (worker *workerT) postStatusOnResponseQ(respStatusCode int, respBody string
 	}
 }
 
-func (worker *workerT) sendRouterResponseCountStat(status *jobsdb.JobStatusT, destination *backendconfig.DestinationT) {
+func (worker *workerT) sendRouterResponseCountStat(status *jobsdb.JobStatusT, destination *backendconfig.DestinationT, errorAt string) {
 	destinationTag := misc.GetTagName(destination.ID, destination.Name)
+	alert := worker.allowRouterAbortedAlert(errorAt)
 	routerResponseStat := stats.Default.NewTaggedStat("router_response_counts", stats.CountType, stats.Tags{
```

Comment:
Should we make sure that we don't erroneously set the `alert` & `errorAt` tags if the status is successful?
```
if status.JobState == jobsdb.Succeeded.State {
   alert = false
   errorAt = ""
}
```

Reply:
`router_response_counts` is the input measurement in `router-aborted-count` definition. 
We should probably judge alert in below way when job succeeds
```go
if status.JobState == jobsdb.Succeeded.State {
  alert = !worker.rt.skipRtAbortAlertForTransformation || !worker.rt.skipRtAbortAlertForDelivery
  errorAt = ""
}
```
WDYT ?

Reply:
Why would the `alert` value ever need to be `true` when job's state is `succeeded`? Do we want to alert on successes too?

Reply:
Basically this `router_response_counts` is a measurement which is being used in `router-aborted-count`. The alert def looks like this
```md
router_aborted_events/router_response_counts
```
where router_response_counts contains events which are sent to router(as a whole success, failure included)
So it would make sense to include alert tag appropriately so that the definition of percentage still remains intact(in terms of `alert` tag)

Basically, if `errorAt` tag is empty, it means that the event was successful & when `alert` tag is `true` it means we would include this event's response while alerting

Reply:
I believe that `errorAt` might be non-empty even if the job's state is `succeeded`, but you do have a point about the rate calculation.

Reply:
Yes, to remove the possibility of success state having an `errorAt`, I have made changes to override `errorAt` with empty string in case of job-success. Can you once verify and let me know if the changes are good ?